### ISSUE TITLE
D401 "not in imperative mood" not truly applicable to @property

### ISSUE THREAD

**Issue by ragazzojp** [PyCQA/pydocstyle#531]
`@property` methods are not used as methods. Their docstrings are used to set the description of the properties. Errors like D401 "docstring not in imperative mood" should not be applicable to methods, at least to getters, that define a `@property`.

IMHO, the following docstring should be accepted for `x`, although not in imperative mood:

```python
@property
def x(self):
    "The X property"
    return self._x
```

Alternatively, please classify these errors differently, so users can switch them off.

**Comment by ragazzojp** [comment]
Same for DAR201, I don't think `:returns:` is mandatory when the property description is a multiline string.

**Comment by samj1912** [comment]
You should be able to use `--ignore-decorators` to ignore pydocstyle errors for certain decorated callables.

http://www.pydocstyle.org/en/stable/usage.html

`DAR201` is not raised by pydocstyle.

**Comment by WasabiFan** [comment]
The above suggested workaround (`--ignore-decorators`) seems to only allow entirely disabling pydocstyle for all `@property` docstrings. I appreciate the availability of a workaround but I think this one isn't sufficiently surgical to effectively unblock a user here.

What _is_ the suggested/intended "mood" for a property docstring otherwise? I guess "Return the X property"?

**Comment by sigmavirus24** [comment]
> but I think this one isn't sufficiently surgical to effectively unblock a user here.

What is blocking a user from your perspective? How should this be more "surgical"?

> What _is_ the suggested/intended "mood" for a property docstring otherwise? I guess "Return the X property"?

How do you document the attributes/properties of classes otherwise?

**Comment by samj1912** [comment]
Fixed by #546
